### PR TITLE
Doc update -upload iso from local

### PR DESCRIPTION
**Pull request by andrijapanicsb**

### CODE REVIEW

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
@shwstppr kindly review

**Comment by andrijapanicsb** [comment]
requesting docbuild

File: source/adminguide/templates.rst [hunk 1]
```diff
@@ -208,10 +208,10 @@ Uploading Templates from a remote HTTP server
...
 vSphere Templates and ISOs
 --------------------------
-
-If you are uploading a template that was created using vSphere Client,
-be sure the OVA file does not contain an ISO. If it does, the deployment
-of VMs from the template will fail.
+.. warning:: 
+      If you are uploading a template that was created using vSphere Client,
+      be sure the OVA file does not contain an ISO. If it does, the deployment
+      of VMs from the template will fail
```
Comment (shwstppr) [inline source/adminguide/templates.rst]:
```suggestion
      of VMs from the template will fail.
```
Not sure if this was deliberate.

File: source/adminguide/templates.rst [hunk 2]
```diff
@@ -275,13 +275,19 @@ To upload a template:
...
 
-It's also possible to upload an already prepared template from your local computer.
-The steps are similar as when Uploading a template from a remote HTTP server, except that you need to choose a local template file from your PC.
+It's also possible to upload an already prepared template or an ISO from your local computer.
+The steps are similar as when Uploading a template/ISO from a remote HTTP server, except that you need to choose a local template/ISO file from your PC.
 For this feature to work, your SSVMs must be supporting HTTPS (for more info please visit `“Using a SSL Certificate for the Console Proxy” 
-<systemvm.html#using-a-ssl-certificate-for-the-console-proxy>`_.)
+<systemvm.html#using-a-ssl-certificate-for-the-console-proxy>`_).
+
+Example GUI dialog of uploading (ISO) from local (browser) is given below:
```
Comment (shwstppr) [inline source/adminguide/templates.rst]:
```suggestion
Example UI dialog of uploading (ISO) from local (browser) is given below:
```
Do we always mention our GUI as UI? And maybe we don't even have mention this line as other parts of doc simply adds screenshots without explicitly mentioning them.

**Comment by nvazquez** [comment]
Thanks @andrijapanic LGTM

**Pull request merged**